src/MovingHarmonicPotentialGPU.cc

```diff
@@ -16,9 +19,9 @@ namespace azplugins
  * \param sysdef System definition
  * \param interf Position of the interface
  */
-ImplicitEvaporatorGPU::ImplicitEvaporatorGPU(std::shared_ptr<SystemDefinition> sysdef,
-                                             std::shared_ptr<Variant> interf)
-    : ImplicitEvaporator(sysdef, interf)
+MovingHarmonicPotentialGPU::MovingHarmonicPotentialGPU(std::shared_ptr<SystemDefinition> sysdef,
+                                                       std::shared_ptr<Variant> interf)
+    : MovingHarmonicPotential(sysdef, interf)
     {
     m_tuner.reset(new Autotuner(32, 1024, 32, 5, 100000, "implicit_evap", m_exec_conf));
```

Comment:
The way that autotuners get created and assigned has changed. See here for an example:

https://github.com/glotzerlab/hoomd-blue/blob/58b7ba503cadd90cc1ed65f40208a8d2de5b7b63/hoomd/md/PotentialExternalGPU.h#L49-L52

The member variable is also going to look different:

https://github.com/glotzerlab/hoomd-blue/blob/58b7ba503cadd90cc1ed65f40208a8d2de5b7b63/hoomd/md/PotentialExternalGPU.h#L39